Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md

```diff
@@ -2211,16 +2213,6 @@ Deletes a Host Set policy.
 | hostSetId | The host set ID. | Required | 
 | policyId | The policy ID. | Required | 
 
-
-#### Context Output
-
-There is no context output for this command.
-#### Command example
-```!fireeye-hx-delete-host-set-policy hostSetId=1005 policyId=YYYXXXYYY```
-#### Human Readable Output
-
->Success
-
```

Comment:
why deleted?

Reply:
Probably happened during the doc review, will add it back